src/webhooks/handleInvoicePaymentFailed.ts

```diff
@@ -22,7 +22,7 @@ export default async function handleInvoicePaymentFailed(
   invoice: Stripe.Invoice,
   objectStorageService: ObjectStorageService,
   paymentService: PaymentService,
-  logger: FastifyLoggerInstance,
+  logger: FastifyBaseLogger,
```

Comment:
Why this change is needed?

Reply:
The change is not needed for this PR, but FastifyLogerInstance is deprecated, so I updated it to the new one.

Reply:
Okay, add the reference link of the Fastify documentation that mentions the deprecation in the description of this PR so we keep track of that in case anything happens.